File: miguel/(bernardo)/Problema0105/problema0105.js
const prompt = require("prompt-sync")({ sigint: true });

//Entrada de valores
console.log("Menu de opções: \n1. Somar dois números. \n2. Raiz quadrada de um número.");
var opcao = parseInteger(prompt("Digite a opção desejada: "));

//Cálculo das operações
if (opcao == 1) {
    var numero1 = parseFloat(prompt("Digite o primerio número: "));
    var numero2 = parseFloat(prompt("Digite o segundo número: "));
    var resultado = numero1 + numero2;
}
else if (opcao == 2){
    var numero1 = parseFloat(prompt("Digite um número: "));
    var resultado = Math.sqrt(numero1);
}
else {
    console.log("Operação não reconhecida");
    var resultado = 0;
}
//Saída 
console.log("O resultado da operação é: " + resultado);

File: miguel/(bernardo)/Problema0105/problema0105.py
import math

# Entrada de valores

print("Menu de opções: \n1. Somar dois números. \n2. Raiz quadrada de um número.")
opcao = int(input("Digite a opção desejada: "))

# Cálculo das operações
if opcao == 1:
    numero1 = float(input("Digite o primerio número: "))
    numero2 = float(input("Digite o segundo número: "))
    resultado = numero1 + numero2
elif opcao == 2:
    numero1 = float(input("Digite um número: "))
    resultado = math.sqrt(numero1)
else:
    print("Operação não reconhecida")
    resultado = 0

# Saída
print(f"O resultado da operação é: {resultado}")
